traits/tests/test_category.py

```diff
@@ -89,15 +89,15 @@ def test_subclass_extension_category(self):
         Seems like the declaration of the subclass (BasePlusPlus) should fail.
         """
         try:
-            x = self.base.pp
+            self.base.pp
             self.fail(
                 msg="base.pp should have thrown AttributeError "
                 "as Category subclassing is not supported."
             )
         except AttributeError:
             pass
 
-        basepp = BasePlusPlus()
+        BasePlusPlus()
```

Comment:
im guessing that this too is a test where we instantiate the object and we expect to see/not-see errors.

Reply:
Yep.